easybuild/easyconfigs/h/Harminv/Harminv-1.4.1-foss-2018a.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'Harminv'
+version = '1.4.1'
+
+homepage = 'https://github.com/stevengj/harminv/blob/master/README.md'
+description = """Harminv is a free program (and accompanying library) to solve the problem of harmonic inversion -
+ given a discrete-time, finite-length signal that consists of a sum of finitely-many sinusoids (possibly exponentially
+ decaying) in a given bandwidth, it determines the frequencies, decay constants, amplitudes, and phases of those
+ sinusoids."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+toolchainopts = {'opt': True, 'unroll': True, 'pic': True, 'cstd': 'c99'}
+
+source_urls = ['https://github.com/stevengj/harminv/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['3ef549e33dfe0bd561c16b8b71702ce3094123e4a4203ed3800653fbe001cb37']
+
+builddependencies = [
+    ('Autotools', '20170619'),
+    ('M4', '1.4.18'),
+    ('pkg-config', '0.29.2'),
+]
+
+#  taken from autogen.sh in root source directory
+preconfigopts = 'autoreconf --verbose --install --symlink --force &&'
+preconfigopts += 'autoreconf --verbose --install --symlink --force &&'
+preconfigopts += 'autoreconf --verbose --install --symlink --force &&'
```

Comment:
I just copied from the autogen.sh:
```
#!/bin/sh

# paranoia: sometimes autoconf doesn't get things right the first time
autoreconf --verbose --install --symlink --force
autoreconf --verbose --install --symlink --force
autoreconf --verbose --install --symlink --force

./configure --enable-maintainer-mode "$@"
```
If I run autogen.sh, it will also run 3 times, plus the config twice. And config without any parameters might fail the configure.